src/chrome/content/rules/League_of_Legends.com.xml

```diff
@@ -94,11 +92,10 @@ Fetch error: http://dominion.leagueoflegends.com/ => https://dominion.leagueofle
 	* Secured by us
 
 -->
-<ruleset name="League of Legends.com (partial)" default_off='failed ruleset test'>
+<ruleset name="League of Legends.com (partial)">
 
 	<target host="leagueoflegends.com" />
 	<target host="signup.leagueoflegends.com" />
-	<target host="support.leagueoflegends.com" />
```

Comment:
The issue is, that there are too many redirects. If you add a rule to redirect this host to `https://support.riotgames.com/` it should work.